step.go

```diff
@@ -330,38 +312,19 @@ The log file is stored in $BITRISE_DEPLOY_DIR, and its full path is available in
 
 	//
 	// Export artifacts
-	var exportedArtifacts []string
-	{
-		fmt.Println()
-		log.Infof("Copy artifacts from Derived Data to %s", absOutputDir)
-
-		proj, err := findBuiltProject(scheme, schemeContainerDir, conf)
-		if err != nil {
-			return ExportOptions{}, fmt.Errorf("failed to open xcproj - (%s), error: %s", absProjectPath, err)
-		}
-
-		customOptions := cfg.XcodebuildAdditionalOptions
-		customOptions = append(customOptions, "-sdk", string(cfg.SimulatorSDK))
```

Comment:
Could remove usages of SimulatorSDK, as it is propably unused after removing this.